action.yaml

```diff
@@ -104,3 +126,23 @@ runs:
       with:
         mask-password: 'true'
         registries: ${{ inputs.AWS_REGISTRY }}
+
+    - name: Get pnpm store directory
+      if: inputs.NODE_PACKAGE_MANAGER == 'pnpm'
+      shell: bash
+      run: |
+        echo "STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
+
+    - uses: actions/cache@v4
+      if: inputs.NODE_PACKAGE_MANAGER == 'pnpm'
+      name: Setup pnpm cache
+      with:
+        path: ${{ env.STORE_PATH }}
+        key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+        restore-keys: |
+          ${{ runner.os }}-pnpm-store-
+
+    - name: Install dependencies
+      if: inputs.NODE_PACKAGE_MANAGER == 'pnpm'
+      shell: bash
+      run: pnpm install
```

Comment:
This action should only setup the env, we leave the project setup to the individual repos

Reply:
That's fair enough, so I'll remove that.